# R Creating the Phyloseq Object

March 1, 2025 - Lily

## Purpose
The purpose of this session is to create a phyloseq object of the OTU table, metadata, taxonomy, and phylogenetic tree. Then, the phyloseq object will be analyzed via processor functions, samples will be rarified, and an alpha rarefaction curve will be generated via R.

## Materials
1. R
2. updated_v2_gastric_cancer_metadata.tsv (email from Tricia)
3. feature-table.txt (email from Tricia)
4. taxonomy.tsv (email from Tricia)
5. tree.nwk (email from Tricia)

## Methods

1. Open a new R project and ensure all required files (see materials) are in the working directory.
2. Load the required packages and data according to the provided Module13_phyloseq.R file, making changes to file paths and names when necessary.
3. Format the OTU table, metadata, taxonomy, and phylogenetic tree according to the codes provided in the Module13_phyloseq.R file. Ensure each data table is saved as a phyloseq variable.
4. Create a collective phyloseq object of the OTU table, metadata, taxonomy, and phylogenetic tree.
5. Analyze the phyloseq object data with processor functions by first filtering the data (e.g. removing non-bacterial sequences and samples with <100 reads), then rarifying samples and generating an alpha rarefaction curve.

## Code
[R Code](/Rscripts/Phyloseq_ObjectScript.R)

## Output Files
1. [/data/project_2/feature-frequency-filtered-table.qza](/QIIME2/exports/feature-frequency-filtered-table.qza)
2. /data/project_2/core-metrics-results-12000
3. [/data/project_2/alpha-rarefaction-filtered.qzv](/QIIME2/visuals/alpha-rarefaction-filtered.png)

## Results
- A phyloseq object of the OTU table, metadata, taxonomy, and phylogenetic tree was created for downstream filtering and/or analysis.
- Alpha rarefaction graph

## Discussion

## Future Directions
- diversity metrics and statistical analysis
